src/dataServices/dynamoDbDataService.ts

```diff
@@ -385,6 +385,25 @@ export class DynamoDbDataService implements Persistence, BulkDataAccess {
         return exportJob;
     }
 
+    async getActiveSubscriptions(params: { tenantId?: string }): Promise<Record<string, any>[]> {
+        this.assertValidTenancyMode(params.tenantId);
+        const subscriptionQuery = DynamoDbParamBuilder.buildGetActiveSubscriptions(params.tenantId);
+        let queryResponse = await this.dynamoDb.query(subscriptionQuery).promise();
+        const subscriptions: Record<string, any>[] = [];
+        do {
+            queryResponse.Items?.forEach((response) => {
+                const item = DynamoDBConverter.unmarshall(response);
+                subscriptions.push({ [item.id]: item });
```

Comment:
why index by `item.id`? I think that we want a simple array of subscriptions i.e. `subscriptions.push(item)`

Reply:
That makes more sense than what I was doing, the return type confused me into thinking we wanted a dictionary of item ids and subscriptions. I've fixed that now